test/chat.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+"""
+Simple CLI-based chat application with LlamaStack agent integration.
+
+This module provides a command-line interface for chatting with an AI agent
+using the LlamaStack client. It includes support for streaming responses,
+session management, and integrated tools.
+"""
+
+import logging
+import os
+import uuid
+from pathlib import Path
+from llama_stack_client import LlamaStackClient
+
+
+# remove logging we otherwise get by default
+logging.getLogger("httpx").setLevel(logging.WARNING)
+
+# Initialize client
+llama_stack_host = os.environ["LLAMASTACK_SERVICE_HOST"]
+client = LlamaStackClient(
+    base_url=f"http://{llama_stack_host}:8321",
+    timeout=120.0,
+)
+
+# Configuration
+model_id = os.environ["LLAMA_STACK_MODELS"].split(",", 1)[0]
+
+########################
+# for now create agent, will not be needed later after
+# agent registration is done as part of the initialization
+# Once that is in place we'll need to look up the agent_id
+prompt_file = Path(__file__).resolve().parent / "prompt.txt"
+system_prompt = open(prompt_file).read()
```

Comment:
Maybe better to use `read_text` or `with`? so the file closing is handled
```suggestion
system_prompt = prompt_file.read_text()
```